telegram/ext/_baseupdateprocessor.py

```diff
@@ -34,32 +34,42 @@ class BaseUpdateProcessor:
     .. versionadded:: NEXT.VERSION
 
     Args:
-        max_concurrent_updates (:obj:`int`): The maximum number of concurrent updates to be
-            processed. If this number is exceeded, new updates will be queued until the number of
-            currently processed updates decreases.
+        max_concurrent_updates (:obj:`int`): The maximum number of updates to be processed
+        concurrently. If this number is exceeded, new updates will be queued until the number
+        of currently processed updates decreases.
 
     Raises:
-        :exc:`ValueError`: If `max_concurrent_updates` is a negative integer.
+        :exc:`ValueError`: If :paramref:`max_concurrent_updates` is a negative integer.
     """
 
     def __init__(self, max_concurrent_updates: int):
         self._max_concurrent_updates = max_concurrent_updates
         if self.max_concurrent_updates < 0:
-            raise ValueError("`max_concurrent_updates` must be a non-negative integer!")
+            raise ValueError("`max_concurrent_updates` must be a positive integer!")
         self._semaphore = BoundedSemaphore(self.max_concurrent_updates or 1)
 
     @property
     def max_concurrent_updates(self) -> int:
         """:obj:`int`: The maximum number of updates that can be processed concurrently."""
         return self._max_concurrent_updates
 
+    @property
+    def semaphore(self) -> BoundedSemaphore:
+        """:class:`asyncio.BoundedSemaphore`: The semaphore used to limit the number of
+        concurrent updates."""
+        return self._semaphore
+
```

Comment:
Is there as specific reason why you added this property? I don't think that we want the user to fiddle with that … 🤔

Reply:
uff did that locally for some purpose, forgot to remove...